### ISSUE TITLE
Add code example for torch.stack()

### ISSUE THREAD

**Issue by onel** [pytorch/pytorch#120303]
### 📚 The doc issue

We are missing some examples for torch.stack()

### Suggest a potential alternative/fix

Add some examples

cc @svekars @brycebortree

**Comment by balasuriyaranganathan** [comment]
Example for torch.stack()
import torch
x = torch.tensor([1,2,3])
y = torch.tensor([4,5,6])

stacked_tensor = torch.stack([x,y],dim=0)
print(stacked_tensor)

**Comment by albanD** [comment]
Feel free to send a PR adding this!
